### ISSUE TITLE
Scan Photos not Working on Synology

### ISSUE THREAD

**Issue by Chengregnarok** [LibrePhotos/librephotos-docker#62]
Hi there this is the newbee trying librephotos on Synology DS720+ again...

I found my scan directory is not working, 

![image](https://user-images.githubusercontent.com/85175548/181916381-1f3bdbd4-18b5-4a80-99c8-44c22c1112f1.png)
now I set /data in frontend
![image](https://user-images.githubusercontent.com/85175548/181916855-528548fd-b286-448a-8623-271418ade8ae.png)
then I put a image file under /data folder

but no matter how many times I tried to scan photos, it leaves 0 photos found
![image](https://user-images.githubusercontent.com/85175548/181917327-901742c6-766a-4eb0-9667-a92751765c6d.png)

and BTW, is there any mothod I can upload my photo from frontend?

**Comment by derneuere** [comment]
What you are seeing is the path from within the docker container. This does not have to map to /data on your Synology. You set the "real path" with the scanDirectory variable.

You can upload images via the frontend. To do that, go to admin area and then activate upload images.

**Comment by Chengregnarok** [comment]
ummm I don't really have that option under admin area
![image](https://user-images.githubusercontent.com/85175548/182025958-4f3996bd-6cd5-43d7-8356-4dc268d0ae53.png)
looks like the .env file didn't set this.. if I need to set this in .env what should I do? below I attached my .env file for your reference
[env.zip](https://github.com/LibrePhotos/librephotos-docker/files/9228832/env.zip)

**Comment by derneuere** [comment]
I have absolutely no idea, which version you are using :D This looks like the UI from a couple of months ago. Which image did you use in your docker-compose.yml

**Comment by Chengregnarok** [comment]
Well... finally I found out why... it looks like the "latest" tag for synology docker is a mystery ....
![Uploading image.png…]()
as shown in the screenshot, the one by default downloaded under "latest" tag supposed to be 2021w34v3.
then when I changed tag setting in .env file to 2022w29, it seems all my problems solved. I didn't try it on my Windows Ubuntu WSL to prove if this is a synology problem only, but I remember this happened before when I try to use docker Home Assistant on my Synology.

My advice is to have some place in frontend showing the version I pulled or installed, that might save some time on finding out where the problem is.
